WrightTools/artists/_base.py

```diff
@@ -20,7 +20,7 @@
 # --- define -------------------------------------------------------------------------------------
 
 
-__all__ = ["Axes", "Figure", "GridSpec", "apply_rcparams"]
+__all__ = ["Axes", "Figure", "GridSpec", "apply_rcparams", "_order_for_imshow"]
```

Comment:
I would tend to avoid adding a private method to `__all__`,

Reply:
you can still import it by name if you need it in another file, `__all__` just affects what gets brought into the namespace with `*` imports/just importing the module